### PR TITLE
Add arm64 binary

### PR DESCRIPTION
**Pull request by koppor**
Fixes https://github.com/awalsh128/cache-apt-pkgs-action/issues/126 with a "quick hack" by adding an arm64 binary - and falling back to it when x86 does not work.

I created the binary using the following command:

    env GOOS=linux GOARCH=arm64 go build -o apt_query-arm64

### CODE REVIEW

File: lib.sh [hunk 1]
```diff
@@ -103,7 +103,7 @@ function get_normalized_package_list {
     | sed 's/[,\]/ /g; s/\s\+/ /g; s/^\s\+//g; s/\s\+$//g' \
     | sort -t' ')
   local script_dir="$(dirname -- "$(realpath -- "${0}")")"
-  ${script_dir}/apt_query normalized-list ${packages}
+  ${script_dir}/apt_query normalized-list ${packages} || ${script_dir}/apt_query-arm64 normalized-list ${packages}
```
Comment (atilsensalduz) [inline lib.sh:106]: I think we need sth like this:

```
  # Determine the architecture
  local architecture=$(dpkg --print-architecture)

  # Use apt_query_arm64 for arm architecture
  if [ "${architecture}" == "arm64" ]; then
    ${script_dir}/apt_query_arm64 normalized-list ${packages}
  else
    ${script_dir}/apt_query normalized-list ${packages}
  fi

```

**Comment by koppor** [comment]
Done. Note that I used a dash instead of an underscore to separate `arm64`.

**Comment by atilsensalduz** [comment]
> Done. Note that I used a dash instead of an underscore to separate `arm64`.

thanks a bunch mate 🚀

**Review by awalsh128 — APPROVED** [review]
Thanks for the PR. Can you provide access to the action yaml so I can dupe the environment? I can put it on the CI so it can get tests before I issue a release.

**Comment by koppor** [comment]
@awalsh128 The full yaml is a bit lengthy: https://github.com/JabRef/jabref/blob/main/.github/workflows/deployment-jdk-ea.yml

The core is to sign up at BuildJet to use their arm runners: https://buildjet.com/for-github-actions

    runs-on: buildjet-2vcpu-ubuntu-2204

**Comment by awalsh128** [comment]
Created #128 to track this and the release.

**Pull request merged**